src/data_info/clients/async_tasks.clj

```diff
@@ -64,17 +82,19 @@
    (paths-async-thread async-task-id jargon-fn end-fn true))
   ([async-task-id jargon-fn end-fn use-client-user?]
    (otel/with-span [s ["paths-async-thread" {:attributes {"async-task-id" (str async-task-id)}}]]
-     (let [{:keys [username] :as async-task} (get-by-id async-task-id)
+     (let [pool (make-threadpool (str async-task-id "-async-tasks-update") 1)
```

Comment:
putting everything through the pool, but having only one thread in said pool, makes the updates async (where desired) but still enforces them getting sent to async-tasks in the right order